docs/Toy/toy/dialects/toy.py

```diff
@@ -191,7 +204,7 @@ class FuncOp(IRDLOperation):
     function_type: FunctionType = attr_def(FunctionType)
     sym_visibility: StringAttr | None = opt_attr_def(StringAttr)
 
-    traits = frozenset((SymbolOpInterface(),))
+    traits = frozenset((SymbolOpInterface(), FuncOpCallableInterface()))
```

Comment:
Do we actually support colls to something different than a Symbol? Why not make FuncOpCallable inherit from SymbolOp?

Reply:
Technically they're not really related, it just so happens that things you call you tend to call by name, but there could be other ways to reference the function. I like that they're decoupled in this way.